plugins/embed-optimizer/class-embed-optimizer-tag-visitor.php

```diff
@@ -60,44 +60,47 @@ public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 			 * for GET requests, as POST requests are not likely to be part of the critical rendering path.
 			 */
 			$preconnect_hrefs = array();
-			if ( true === $processor->has_class( 'wp-block-embed-youtube' ) ) {
+			$has_class        = static function ( string $wanted_class ) use ( $processor ): bool {
+				return true === $processor->has_class( $wanted_class );
+			};
```

Comment:
Is this just to reduce repetition below, or actually because of PHPStan? 🤔

Reply:
Ah yes, just to reduce repetition.